dacapo/experiments/tasks/post_processors/watershed_post_processor.py

```diff
@@ -1,5 +1,4 @@
 from dacapo.experiments.datasplits.datasets.arrays import ZarrArray
-from dacapo.store.array_store import LocalArrayIdentifier
```

Comment:
This needs to be included in new version, I think...

Reply:
This still missing?

Reply:
May need to wrap with this: 
```
from typing import TYPE_CHECKING, Iterable

if TYPE_CHECKING:
    from dacapo.store.local_array_store import LocalArrayIdentifier
    from dacapo.experiments.tasks.post_processors import (
        ThresholdPostProcessorParameters,
    )

```

Reply:
We made LocalArrayIdentifier importable now, so I *could* just remove all of the quoted versions instead? Adding an if around type checking should probably be a global change imo